microshift_release_notes/microshift-4-12-release-notes.adoc

```diff
@@ -6,15 +6,69 @@ include::_attributes/attributes-microshift.adoc[]
-{product-title} is now available as a Developer Preview. Features and known issues that pertain to {product-title} {product-version} are included in this topic.
+{product-title} is now available as a Developer Preview software. Features and known issues that pertain to {product-title} {product-version} are included in this topic. For more information about the support scope of Red Hat Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
 
-{product-title} {product-version} is supported on {op-system-first} 4.12 and {op-system-base-full} 8.4 and 8.5.
+[IMPORTANT]
+====
+Red Hat will not provide or support an update or upgrade path from Developer Preview and Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
+====
+
+[id="microshift-known-issues"]
+== Known issues
+* There is an issue exporting services via the Route API when the host and routerCanonicalHostname do not include the sub domain name. To work around this problem, use the subdomain setting when creating a Route.
++
+.Example
+[source, terminal]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: route-1
+spec:
+  subdomain: svc-subdomain  ⇐ specify the subdomain
+  port:
+    targetPort: 8080
+  to:
+    kind: Service
+    name: svc-1
+----
+
+* There is an issue resuming pods when a service is idled. Unidling is not supported by the {product-title} software, and manual re-scaling the replicas is required.
+
+* It is possible to over-provision the storage system by creating persistent volumes (PVs) that do not actually fit on the disk when using the `spare-gb` configuration value in `lvmd.conf`. To protect capacity from being used by the `topoLVM` driver, use a new separate volume group (VG) that is not exposed to topolvm.
+
+* There is a known issue involving `runc` and `systemd`. `Runc` is overly verbose about `mount ops`. These messages are innocuous and do not indicate a problem with the system's operation.  This bug affects any Linux platform running a container runtime built on `runc`.
++
+You can suppress all `runc` mount logs through `systemd`. Create the file `/etc/systemd/system/run-containerd-.mount.d/10-silence.conf`, or whatever prefix you need to silence, with:
++
+[source, terminal]
+----
+[Mount] LogLevelMax=notice
+----
++
+And reload the daemon.
+
+* {product-title} uses long-lived certificates but some of the code imported from {ocp} prefers longer lifetimes and logs warning messages. The messages are innocuous and do not indicate a problem with the system's operation.
+
+* There is an issue with the way {product-title}'s CNI driver manages the firewall rules that means if the firewall settings are modified after {product-title} is started then {product-title} must be restarted. The recommended workaround is to configure firewall rules before starting {product-title}.
```

Comment:
Something about _that means_ bugs me. Maybe _such that, if the firewall settings are modified..._